#! /usr/bin/env python
# Task Long configuration

PACKAGE='floor_nav'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *
from task_manager_lib.parameter_generator import *

gen = TaskParameterGenerator()
#       Name                    Type            Description
#       Default    Min   Max
gen.add("goal_x",        double_t, 0,    "X coordinate of destination",  0.)
gen.add("goal_y",        double_t, 0,    "Y coordinate of destination",  0.)
gen.add("goal_theta",    double_t, 0,    "Heading at destination",  0.)
gen.add("dist_threshold",double_t, 0,    "Distance at which a the target is considered reached",  0.1)

exit(gen.generate(PACKAGE, "floor_nav", "TaskPlanTo"))

